pgactivity/ui.py

```diff
@@ -156,7 +156,7 @@ def main(
                 else:
                     pg_procs.reset()
                     ui.end_interactive()
-                    changes = {
+                    changes: types.Changes = {
```

Comment:
This change is also unrelated to the new feature, but it's probably due to a new Mypy version (more strict) which we didn't test before (I'm seeing the error as well on master).

In fact, I'd simply suggest:
```suggestion
                    changes: dict[str, Any] = {
```

If you can commit this in a distinct commit, that's also be better.

Reply:
Yes it's due to mypy.

Reply:
```
pg_activity on  xmin_425 [$!?⇕] via 🐍 v3.13.2 (.venv)
❯ mypy ./pgactivity/
pgactivity/ui.py:164: error: Incompatible types in assignment (expression has type "float", target has type "int")  [assignment]
pgactivity/ui.py:169: error: Incompatible types in assignment (expression has type "QueryMode", target has type "int")  [assignment]
pgactivity/ui.py:174: error: Incompatible types in assignment (expression has type "SortKey", target has type "int")  [assignment]
Found 3 errors in 1 file (checked 17 source files)
```

Reply:
It's in a separate commit.